docs/crr-pause-resume.md

```diff
@@ -0,0 +1,54 @@
+# Cross-Region Replication (CRR) Manual Pause and Resume
+
+## Description
+
+This feature offers a way for users to manually pause and resume cross-region
+replication (CRR) operations by site.
+
+## Design
+
+A RESTful API will expose methods for users to pause and resume cross-region
+replication operations.
+
+Redis offers a pub/sub function. We will utilize this function to propagate
+requests to all active CRR Kafka Consumers on all nodes with backbeat
+containers setup for replication.
+
+We want to pause and resume the CRR service at the lowest level (in our case,
+pause and resume all Kafka Consumers subscribed to the CRR topic). We want to
+perform these actions at the lowest level in order to stop processing any
+current batch jobs.
```

Comment:
Does this stop jobs that are currently in process of being replicated (i.e. multipart uploads).

Reply:
No, once the consumer "consumes" the job, the job has already been sent to the queue for completing the task. So I'm afraid it won't stop the currently processing jobs

Reply:
That seems fine to me 😄. I just thought the wording here made it sound as such. Particularly 'stop processing any current batch jobs'.

Reply:
ah good to know, I'll update the wording here